src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -171,11 +171,13 @@ trait FastSync {
 
     syncStateStorageActor ! fastSyncStateStorage
 
+    private var blockChainOnlyPeers = Set.empty[Peer]
```

Comment:
it is additional logic, we have limit on this pool set to 100 in conf, new entries will naturally remove olde ones (including disconnected)